stac_ipyleaflet/stac_discovery/stac.py

```diff
@@ -311,10 +333,10 @@ def get_tile_url(
         Returns:
             str: Returns the STAC Tile layer URL.
         """
-        if url is None and collection is None:
+        if url is None and collection is None: # QUESTION: should this be an "and" or "or" given error statement ?
```

Comment:
I think this is correct as is - `if` _**both are none**_ `then` **we don't have either url or collection**.

Reply:
👍🏼 will update `raise ValueError("Either url or collection must be specified. stac_tile")` to say `and` then